native/membrane_videocompositor_opengl_rust/glad-gles2/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
it would be nice to put w README here documenting how to regenerate the bindings